src/js/components/award/financialAssistance/CFDAViz.jsx

```diff
@@ -97,7 +100,34 @@ export default class CFDAViz extends React.Component {
     content = () => {
         const { view } = this.props;
         if (view === 'table') return (<CFDATable {...this.props} />);
-        if (view === 'single' || !view) return (<SingleCFDA data={this.props.cfda} />);
+        if (view === 'single' || !view) return (<SingleCFDA currentCfda={this.props.cfda} />);
+        return null;
+    }
+
+    title = () => {
+        const {
+            cfda,
+            view,
+            allCFDAs,
+            onDropdownClick
+        } = this.props;
+        if (view === 'single' || !view) {
+            if (allCFDAs.length > 1) {
+                const options = cloneDeep(allCFDAs)
+                    .sort((a, b) => parseFloat(a.cfdaNumber) - parseFloat(b.cfdaNumber))
+                    .map((x) => ({
+                        name: `${x.cfdaNumber} ${x.cfdaTitle}`,
+                        onClick: onDropdownClick
+                    }));
+                return (<Picker
+                    options={options}
+                    dropdownDirection="right"
+                    backgroundColor="#215493"
+                    selectedOption={`${cfda.cfdaNumber} ${cfda.cfdaTitle}`}
+                    sortFn={() => 1} />);
```

Comment:
why?

Reply:
That is a great question it should be this
```
(a, b) => parseFloat(a.cfdaNumber) - parseFloat(b.cfdaNumber);
```
I think we should have a discussion with design on how it should look.